### PR TITLE
FNA-15: ENHANCEMENT: Search 

### PR DESCRIPTION
**Pull request by alexn-spire**
Updated empty results cell to show proper graphic and text.

### CODE REVIEW

File: Formed/API.swift [hunk 1]
```diff
@@ -153,12 +152,53 @@ class APIController {
...
             urlComponents.path = "/api/v1/public/pages/books/"
             return urlComponents.url
         default:
             return nil
         }
     }
+    
+    func searchURL(for text: String, limit: Int, type: MediaType) -> URL? {
+        urlComponents.host = "ai-search-dev.formed.org"
+        switch type {
```
Comment (gpasq) [inline Formed/API.swift]: Hostnames in config file.  See similar comment in other PR.

File: Formed/CategoryDetailBookCell.swift [hunk 2]
```diff
@@ -27,8 +27,10 @@ class CategoryDetailBookCell: UITableViewCell {
             return
         }
         
+    
+        self.leftImgView.layoutIfNeeded()
         let frame = self.leftImgView.frame
-        let filter = AspectScaledToFitSizeFilter(size: CGSize(width: frame.width, height: frame.height))
+        let filter = AspectScaledToFillSizeFilter(size: CGSize(width: frame.width, height: frame.height))
```
Comment (gpasq) [inline Formed/CategoryDetailBookCell.swift]: This might stretch and muck up aspect ratio, so test as needed.

File: Formed/SearchOperation.swift [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+import Foundation
+import UIKit
+import Alamofire
+import SwiftyJSON
+import RealmSwift
+
+class SearchOperation {
+    
+    class func search(for text: String, type: MediaType, completionHandler: @escaping (Result<Array<MediaItem>>) -> Void) {
+        let api = APIController()
```
Comment (gpasq) [inline Formed/SearchOperation.swift]: I haven't seen where this is called, but make sure you're firing all four searches simultaneously.

Comment (alexn-spire) [inline Formed/SearchOperation.swift]: It's called in SearchVC.swift, and it is all indeed firing simultaneously.

**Pull request closed without merging**